raiden/raiden_service.py

```diff
@@ -922,17 +922,6 @@ def _initialize_messages_queues(self, chain_state: ChainState):
         for queue_identifier, _event_queue in events_queues.items():
             self.start_health_check_for(queue_identifier.recipient)
 
-            # for event in event_queue:
```

Comment:
not sure why this was commented in the first place.

if anyone knows, please fill me in on this

Reply:
there is already a task for that commented code, we can remove it